packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockFrameRateRange.swift

```diff
@@ -2,20 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import camera_avfoundation
+@testable import camera_avfoundation
 
 // Import Objective-C part of the implementation when SwiftPM is used.
 #if canImport(camera_avfoundation_objc)
   import camera_avfoundation_objc
 #endif
 
-/// A mock implementation of `FLTFrameRateRange` that allows mocking the class properties.
-final class MockFrameRateRange: NSObject, FLTFrameRateRange {
-  var minFrameRate: Float
-  var maxFrameRate: Float
+/// A mock implementation of `FrameRateRange` that allows mocking the class properties.
+final class MockFrameRateRange: NSObject, FrameRateRange {
+  var minFrameRate: Float64
+  var maxFrameRate: Float64
```

Comment:
why is this change required?

Reply:
To match the `FrameRateRange` protocol, which was updated to match `AVFrameRateRange`